lib/sender.ts

```diff
@@ -90,10 +90,13 @@ export class Sender extends Link {
   /**
    * Sends the message
    * @param {Message | Buffer} msg The message to be sent. For default AMQP format msg parameter
-   * should be of type Message interface. For a custom format, the msg parameter should be a Buffer.
+   * should be of type Message interface. For a custom format, the msg parameter should be a Buffer
+   * and a valid value should be passed to the `format` argument.
    * @param {Buffer | string} [tag] The message tag if any.
-   * @param {number} [format] The message format. Usually it is zero. Specify this
-   * if a message with custom format needs to be sent.
+   * @param {number} [format] The message format. Specify this if a message with custom format needs
+   * to be sent. For example, sending multiple messages in a single batch needs the format `0x80013700`.
```

Comment:
Should clarify that 0x80013700 is an Azure/ServiceBus specific batching format.

Reply:
Removed Azure specific code and added AMQP specific info only